packages/paste-website/src/component-examples/ElevationFoundationExamples.tsx

```diff
@@ -0,0 +1,315 @@
+import { ImageCaption } from "../components/ImageCaption";
+
+export const ElevatedBackgroundColorTokensExample = `
+const ElevatedBackgroundColorTokens = () => {
+  const TokensTemplate = ({
+    backgroundColor,
+    backgroundColorElevated,
+    hasBorder,
+    backgroundColorBase,
+    backgroundColorElevation,
+  }) => {
+    return (
+      <Box>
+        <Box position="relative">
+          <Box
+            backgroundColor={backgroundColor}
+            borderRadius="borderRadius30"
+            height="163px"
+            width="148px"
+            borderWidth={hasBorder ? "borderWidth10" : "borderWidth0"}
+            borderStyle="solid"
+            borderColor="colorBorderWeaker"
+          />
+          <Box
+            backgroundColor={backgroundColorElevated}
+            borderRadius="borderRadius30"
+            height="163px"
+            width="148px"
+            position="absolute"
+            top={35}
+            left={35}
+          />
+        </Box>
+        <Box marginTop="space200" fontSize="fontSize30" fontWeight="fontWeightSemibold">
+          <Text as="p" color="colorTextWeak">
+            Base:
+          </Text>
+          <Text as="p" marginTop="space30">
+            {backgroundColorBase}
+          </Text>
+          <Text as="p" marginTop="space40" color="colorTextWeak">
+            Elevation:
+          </Text>
+          <Text as="p" marginTop="space30">
+            {backgroundColorElevation}
+          </Text>
+        </Box>
+      </Box>
+    );
+  };
+  return (
+    <Box display="flex" flexWrap="wrap" columnGap="space70" rowGap="space90">
+      <TokensTemplate
+        backgroundColor="colorBackgroundBody"
+        backgroundColorElevated="colorBackgroundBodyElevation"
+        backgroundColorBase="$color-background-body"
+        backgroundColorElevation="$color-background-elevation-body"
+        hasBorder
+      />
+      <TokensTemplate
+        backgroundColor="colorBackgroundStrong"
+        backgroundColorElevated="colorBackgroundElevation"
+        backgroundColorBase="$color-background-body"
+        backgroundColorElevation="$color-background-elevation"
+      />
+      <TokensTemplate
+        backgroundColor="colorBackgroundInverse"
+        backgroundColorElevated="colorBackgroundInverseElevation"
+        backgroundColorBase="$color-background-body"
+        backgroundColorElevation="$color-background-elevation-inverse"
+      />
+      <TokensTemplate
+        backgroundColor="colorBackgroundInverseStrong"
+        backgroundColorElevated="colorBackgroundInverseStrongElevation"
+        backgroundColorBase="$color-background-body"
+        backgroundColorElevation="$color-background-elevation-inverse-strong"
+      />
+    </Box>
+  );
+};
+
+render(
+  <ElevatedBackgroundColorTokens />
+)
+`.trim();
+
+export const ShadowTokensLevel1Example = `
```

Comment:
suggestion:

- Remove "(accent)" from all text in the ShadowTokensLevel... examples.
- For the examples in twilio-dark theme, make the text "Shadow in twilio-dark theme"